src/main/java/com/lazerycode/jmeter/mojo/ConfigureJMeterMojo.java

```diff
@@ -508,9 +507,6 @@ private void extractConfigSettings(Artifact artifact) throws IOException {
 				// Only interested in files in the /bin directory that are not properties files
 				if (!jarFileEntry.isDirectory() && jarFileEntry.getName().startsWith("bin") && !jarFileEntry.getName().endsWith(".properties")) {
 					File fileToCreate = new File(jmeterDirectory, jarFileEntry.getName());
-					if (jarFileEntry.getName().endsWith(logConfigFilename) && fileToCreate.exists()) {
-						break;
-					}
 					copyInputStreamToFile(configSettings.getInputStream(jarFileEntry), fileToCreate);
```

Comment:
Looks like we are no longer using the default logkit.xml supplied with JMeter, has this been removed in 3.2?